book/src/developer-guides/frame-nft.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+Frame-NFT is aimed at developers to provide them with a reliable and adjustable codebase for NFT-based projects. 
+This is especially necessary due to the various forms that NFTs can present as, each with their own unique traits.
```

Comment:
This sentence is a bit weird to me, perhaps not needed here

Reply:
With suggestion above in mind, it is a loose thread, will be cut.